### DISCUSSION TITLE
403 error when updating from 2.15.0 to 2.16.9

### DISCUSSION THREAD

**Discussion by karthiksw** [quarkusio/quarkus#35184] [Community]
We have been running application with  (artifactId)quarkus-universe-bom version 2.15.0.Final. This quarkus-vertx-http jar  has Vulnerability issue(CVE-2023-2974) in application.Currently application running with quarkus-vertx-http jar file version 2.15.0.Final. I tried to update quarkus-universe-bom jar with latest jar file version 2.16.9.Final 
While access application, API(POST method) is giving 403 error. Other get API is working with same jar. 
Need help and advice about this issue?

**Comment by gsmet** [comment]
I don't think the title of your discussion is accurate, I'm going to change it.

Then you need to provide a lot more details such as your configuration, your extensions, how your endpoint is secured... Ideally, you should provide a simple reproducer that works with 2.15.0.Final and doesn't with 2.16.9.Final.

/cc @sberyozkin

**Comment by karthiksw** [comment]
Do you need pom.xml and application.properties?
We have intermediator services which is used from UI to make API call. Intermediator services is fuse-springboot- application. This facade acts as the routing and transformation component mediating calls in between microservices.

 UI---Invoke API----->intermediator service--------invoke back-end API-------->WordPress Application (- we have used quarkus jar in this application ).
